pkg/components/component.go

```diff
@@ -54,20 +54,39 @@ type Component interface {
 	IsUpdatable() bool
 }
 
-type componentBase struct {
+type labellable struct {
 	labeller *labeller.Labeller
-	ytsaurus *apiproxy.Ytsaurus
 }
 
-func (c *componentBase) GetName() string {
+func (c *labellable) GetName() string {
 	return c.labeller.ComponentName
 }
 
-func (c *componentBase) GetLabel() string {
+func (c *labellable) GetLabel() string {
 	return c.labeller.ComponentLabel
 }
 
-func (c *componentBase) SetReadyCondition(status ComponentStatus) {
+type localComponent struct {
```

Comment:
Also, a couple of comments (a-la the ones from the PR description) would look nice here.

Reply:
done